src/Home/HomePage.tsx

```diff
@@ -55,7 +50,9 @@ const HomePage = () => {
             <h1 className='title-blog'>Latest news</h1>
             <Blog />
           </section>
-          <footer className='container-fotter'>stopka</footer>
+          <footer className='container-fotter'>
```

Comment:
what I said about cases like this? you should add <footer> tag inside Footer component to wrap all the footer content. And there just use `<Footer />`. One more thing, you should change styling - remove class names and add the module css styling instead